.github/workflows/ci.yaml

```diff
@@ -139,16 +138,16 @@ jobs:
           set -e
```

Comment:
Let's add pipefail here to also detect non-zero exit codes of the golem worker invoke command itself

Reply:
Alright I'm going to change this

Reply:
Added now!